server/src/router.js

```diff
@@ -1,6 +1,11 @@
 const router = require('express').Router();
-const loginValidation = require('./controllers/validation/loginValidation');
 
-router.post('/login', loginValidation);
-module.exports = router;
+const {
+  validateLogin,
+  checkEmailExist,
+  checkPassword,
+} = require('./controllers');
+
+router.post('/login', validateLogin, checkEmailExist, checkPassword);
```

Comment:
These 3 are actually one controller, so combine them and try to clean up the code